src/PrestaShopBundle/Controller/Admin/ModuleController.php

```diff
@@ -325,23 +325,35 @@ public function importModuleAction(Request $request)
         $moduleManager = $this->get('prestashop.module.manager');
         try {
             $file_uploaded = $request->files->get('file_uploaded');
-            $violations = $this->get('validator')->validate($file_uploaded, new File(
-                [
+            $notNullViolations = $this->get('validator')->validateValue(
+                $file_uploaded,
+                new Assert\NotNull(array(
+                        'message' => $this->get('translator')->trans(
+                            'No file has been received. The server may have refused it because of its size.',
+                            array(),
+                            'Admin.Modules.Notification'))));
+
+            $fileViolations = $this->get('validator')->validateValue($file_uploaded, new Assert\File(
+                array(
+                    'disallowEmptyMessage' => 'An empty file is not allowed.',
```

Comment:
This line will be removed. This was a test we made together, to know why an empty file is not properly handled by the validator.